src/core/Hocon/Akka.Hocon.csproj

```diff
@@ -1,10 +1,14 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.4;net461</TargetFrameworks>
+    <TargetFrameworks>net45;netstandard1.4</TargetFrameworks>
```

Comment:
Why do you want to target netstandard1.6 when it also works with 1.4 ? I think Microsoft recommend using the lowest possible version for maximum compatibility.